src/utils/uint256.cairo

```diff
@@ -19,3 +33,40 @@ func uint256_exp{range_check_ptr}(a: Uint256, b: Uint256) -> Uint256 {
     let (res, _) = uint256_mul(a, pow);
     return res;
 }
+
+// @notice Performs sign extension on a Uint256 number.
+// @param x The number to be sign extended.
+// @param b The size in bytes minus one of x to consider.
+// @return The result of sign extending x from position b.
+// If b > 31, the result is x. Otherwise, the sign bit is extended from position b.
+func uint256_signextend{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(
+    x: Uint256, b: Uint256
+) -> Uint256 {
+    alloc_locals;
+    // If b > 31, then the result is x.
```

Comment:
the comment says

"if b > 31" so "if not (b <= 32)"

but the code does

"if not (b < 32)"

and writing this comment I'm doubting about it in the meantime. that's why I don't like small comment for obvious operations

Reply:
so:
- the comment should be `if b is not strictly smaller than 32, we return x`, otherwise it's confusing 
- or the comment should not exist

i'll delete the comment